packages/cms-base-layer/components/SwProductReviews.vue

```diff
@@ -42,15 +48,21 @@ const reviewsList = computed<Schemas["ProductReview"][]>(
 
 const format: Intl.DateTimeFormatOptions = {
   year: "numeric",
-  month: "short",
+  month: "numeric",
   day: "numeric",
   hour: "numeric",
   minute: "numeric",
-  hour12: true,
 };
 
-const formatDate = (date: string) =>
-  new Date(date).toLocaleDateString("en-us", format);
+const { sessionContext } = useSessionContext();
+const navLang =
```

Comment:
localeCode from the session is not the best source. I saw that in many examples, `en` code was used for `en-GB` and `en-US`

Reply:
do you mean session? This is preventing from hardcoded locales, session now contains new `languageInfo. localeCode` which exactly indicates the locale. What is your proposition here?